google/cloud/storage/examples/storage_object_acl_samples.cc

```diff
@@ -153,6 +153,55 @@ void UpdateObjectAcl(gcs::Client client, int& argc, char* argv[]) {
   (std::move(client), bucket_name, object_name, entity, role);
 }
 
+void PatchObjectAcl(gcs::Client client, int& argc, char* argv[]) {
+  if (argc != 5) {
+    throw Usage{"patch-object-acl <bucket-name> <object-name> <entity> <role>"};
+  }
+  auto bucket_name = ConsumeArg(argc, argv);
+  auto object_name = ConsumeArg(argc, argv);
+  auto entity = ConsumeArg(argc, argv);
+  auto role = ConsumeArg(argc, argv);
+  //! [patch object acl]
+  namespace gcs = google::cloud::storage;
+  [](gcs::Client client, std::string bucket_name, std::string object_name,
+     std::string entity, std::string role) {
+    gcs::ObjectAccessControl original_acl =
+        client.GetObjectAcl(bucket_name, object_name, entity);
+    auto new_acl = original_acl;
+    new_acl.set_role(role);
+    gcs::ObjectAccessControl acl =
```

Comment:
`updated_acl` instead of `acl` to follow `original_acl` and `new_acl`.

Reply:
Fixed.

Reply:
Original comment was with respect to `gcs::ObjectAccessControl acl` and it should be updated to `gcs::ObjectAccessControl updated_acl` instead of variable name `new_acl`.

Reply:
Doh!  Fixed for realsies I think.